perception/detected_object_validation/src/object_lanelet_filter.cpp

```diff
@@ -23,6 +24,22 @@
 
 #include <lanelet2_core/geometry/Polygon.h>
 
+namespace
+{
+geometry_msgs::msg::Point32 transformFootprintPoint(
+  const geometry_msgs::msg::Point32 & point, const geometry_msgs::msg::Pose & pose)
+{
+  tf2::Transform tf_data_;
```

Comment:
@yukkysaito
Thank you for your advise!
I used transformPoint in https://github.com/autowarefoundation/autoware.universe/commit/35fa236fb5c5c2e99368165d4adf01fabfa16bb7, so that code you advised is outdated.